spec/comm.mediawiki

```diff
@@ -182,13 +182,17 @@ connection is established, the client will supply their account ID and signature
 
 If a client unexpectedly disconnects with active orders, the orders may match in
 the client's absence. A list of any pending matches is included in the response.
+If the client has a broken rule, they will not be able to trade for the duration
+of their penalization.
 
 <code>result</code>
 {|
 ! field   !! type   !! description
 |-
 | matches || &#91;object&#93; || list of [[orders.mediawiki/#Match_negotiation|Match objects]]
 |-
+| penalty || object || the client's most recent [[community.mediawiki/#Rules_of_Community_Conduct|broken rule]]. omitted if in good standing
```

Comment:
If this is going to be an object, the object will need to be defined. Can you define the object separately and just reference it here and in the `penalty` notification?

Reply:
Is this how you mean?

Reply:
Yes, but the `sig` would not be a field in the object. It would stay as a field in the `penalty` payload.